doc/moderation.rst

```diff
@@ -45,6 +45,17 @@ Code reviews
 When reviewing code it is highly recommended to check whether it complies
 with official style guide and point out mismatches.
 
+Prepared replies
```

Comment:
More replies need to be added here, for now I only added one that links to the commit message guidelines